sw/banshee/src/main.rs

```diff
@@ -137,6 +144,30 @@ fn main() -> Result<()> {
                 .multiple(true)
                 .help("Pass command line arguments to LLVM"),
         )
+        .arg(
+            Arg::with_name("train-data-bin-file-path")
+                .long("train-data-bin-file-path")
+                .takes_value(true)
+                .help("Path to the binary file of the data for training."),
+        )
+        .arg(
+            Arg::with_name("train-data-mem-offset")
+                .long("train-data-mem-offset")
+                .takes_value(true)
+                .help("Define the offset at which the train data should be written into DRAM."),
+        )
+        .arg(
+            Arg::with_name("train-labels-bin-file-path")
+                .long("train-labels-bin-file-path")
+                .takes_value(true)
+                .help("Path to the binary file of the labels for training."),
+        )
+        .arg(
+            Arg::with_name("train-labels-mem-offset")
+                .long("train-labels-mem-offset")
+                .takes_value(true)
+                .help("Define the offset at which the train labels should be written into DRAM."),
+        )
```

Comment:
Can we maybe generalize this to not be specific to training data and labels? We could provide a list of preloadable binary files in the style `--bin-files <file>:<addr>,<file>:<addr>,..` or something along these lines.